packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -443,19 +447,28 @@ async function notMemoizedbase64({ file, args = {}, reporter }) {
     return null
   }
 
+  const cacheKey = base64CacheKey(file.internal.contentDigest, options)
+  // Not all tranformer plugins are going to provide this parameter hence the fallback
```

Comment:
Wouldn't we want to memoize still as a backup?

Reply:
just so i understand you mean memoizing the whole function just in case cache is not available right?